patches/all/@oclif+dev-cli+1.22.2.patch

```diff
@@ -248,14 +248,13 @@ index 343eb00..5521e2d 100644
 +    console.log(`fetchNodeBinary using tmp="${tmp}`);
 +
      if (arch === 'arm')
-         arch = 'armv6l';
+         arch = 'armv7l';
```

Comment:
Okay... We don't currently use this, right? Oclif installers for ARM. Just checking.

Reply:
It's an upstream bugfix for the urls to download from, from https://nodejs.org/en/blog/release/v12.18.3/ they look like 
```
ARMv7 32-bit Binary: https://nodejs.org/dist/v12.18.3/node-v12.18.3-linux-armv7l.tar.xz
```